ballet/validation/__init__.py

```diff
@@ -58,7 +58,8 @@ def get_accepted_features(features, proposed_feature):
         list[Feature]: list of features with the proposed feature not in it.
 
     Raises:
-        Error: Could not deselect exactly the proposed feature.
+        ballet.exc.BalletError: Could not deselect exactly the proposed
+            feature.
     """
     def neq(feature):
```

Comment:
Why neq instead of eq?

I like that we're putting this in a function so that it's easier to weed out if we ever find a better solution to feature equality. I would recommend changing all instances of source comparing later on.

Reply:
good point, `eq` is clearer; and then we can take the complement of eq to deselect features. fixing.